include/dfrn.php

```diff
@@ -2053,10 +2053,10 @@ private function process_entry($header, $xpath, $entry, $importer) {
 			if (($item["network"] != $author["network"]) AND ($author["network"] != ""))
 				$item["network"] = $author["network"];
 
-			if($importer["rel"] == CONTACT_IS_FOLLOWER) {
-				logger("Contact ".$importer["id"]." is only follower. Quitting", LOGGER_DEBUG);
-				return;
-			}
+			//if($importer["rel"] == CONTACT_IS_FOLLOWER) {
```

Comment:
Can you add a simple comment why this was commented out. so we we (or maybe other people) don't have to remember or ask for the reason and can better decide at some future time if it can be deleted